### PR TITLE
[17.0] [MIG] product_pricelist_supplierinfo: Migration to 17.0

### PR DESCRIPTION
**Pull request by mdurepos**

### CODE REVIEW

**Comment by suniagajose** [comment]
hi @luisg123v

can you review and merge this please?

thanks,

File: product_pricelist_supplierinfo/README.rst [hunk 1]
```diff
@@ -17,23 +17,24 @@ Supplier info prices in sales pricelists
     :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
     :alt: License: AGPL-3
 .. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fproduct--attribute-lightgray.png?logo=github
-    :target: https://github.com/OCA/product-attribute/tree/16.0/product_pricelist_supplierinfo
+    :target: https://github.com/OCA/product-attribute/tree/17.0/product_pricelist_supplierinfo
```
Comment (luisg123v) [inline product_pricelist_supplierinfo/README.rst:20]: This is in the migration commit, but it should be on a separate commit that includes precommit changes. Presentation should be:
- Module history from the previous version (several commits
- A commit running precommit, named `[IMP] module_name: pre-commit auto fixes`
- The actual migration commit. Here you should remove `[17.0]`, this should only be included in PR's title

File: product_pricelist_supplierinfo/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# Copyright 2018 Tecnativa - Vicent Cubells
+# Copyright 2018 Tecnativa - Pedro M. Baeza
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    "name": "Supplier info prices in sales pricelists",
+    "summary": "Allows to create priceslists based on supplier info",
+    "version": "17.0.1.0.1",
```
Comment (luisg123v) [inline product_pricelist_supplierinfo/__manifest__.py]:
```suggestion
    "version": "17.0.1.0.0",
```

**Review by luisg123v — CHANGES_REQUESTED** [review]

File: product_pricelist_supplierinfo/README.rst (hunk 1 above)
Comment (mdurepos) [reply product_pricelist_supplierinfo/README.rst:20]: @luisg123v thanks for the review. I'll correct and re-push when I get some time.

**Comment by suniagajose** [comment]
hi @luisg123v 

can you check with someone at OCA why the bot did not allow you to merge it?

thanks,

**Comment by luisg123v** [comment]
@suniagajose I didn't try to merge, I just tried to mark it as a migration PR

**Comment by suniagajose** [comment]
ok @luisg123v 

whom can i ask to mark as migration instead?

**Comment by luisg123v** [comment]
Hi @sbidoul, @pedrobaeza,

I tried to mark this module as a migration one on https://github.com/OCA/product-attribute/pull/1632#issuecomment-2164171292, but it failed with a permission error, despite being a maintainer. Besides, I noticed I wasn't notified for review.

Is there any issue with the bot you are aware of?

Regards,

CC @desdelinux

**Comment by pedrobaeza** [comment]
Please split the pre-commit initial changes from the migration commit as stated in https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-17.0

**Comment by sbidoul** [comment]
@luisg123v 

> I tried to mark this module as a migration one on https://github.com/OCA/product-attribute/pull/1632#issuecomment-2164171292, but it failed with a permission error, despite being a maintainer.

I don't think the `ocabot migration` command checks for maintainers. Wondering if the bot could detect migration PRs automatically.

> Besides, I noticed I wasn't notified for review.

https://github.com/OCA/oca-github-bot/issues/193 Seems there is an open PR, but it's a big diff to review.

**Comment by pedrobaeza** [comment]
> I don't think the ocabot migration command checks for maintainers. Wondering if the bot could detect migration PRs automatically.

That's a very difficult task, as there's nothing characteristic that can be used for that. The only thing may be the PR title, and people is not very standard with it... so it will be the same time to ask a contributor to change the PR title (and to detect such changes from the bot), than launching `ocabot migrat...` command.

**Comment by suniagajose** [comment]
> Please split the pre-commit initial changes from the migration commit as stated in https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-17.0

Hi @mdurepos 

Can you do it, please?

Regards,

**Comment by mdurepos** [comment]
@suniagajose I just force pushed with the auto-fix commit and real manual changes separated. I believe I caught everything from the previous comments.

**Comment by suniagajose** [comment]
thanks @mdurepos 

@moylop260 @pedrobaeza i think now is ready....
